catalog.yml

```diff
@@ -164,7 +164,8 @@
   preprint_citation: doi:10.1101/2021.03.04.433874
 - repo_url: https://github.com/lotusnprod/lotus-manuscript
   html_url: https://lotusnprod.github.io/lotus-manuscript
-  preprint_citation: doi:10.1101/2021.02.28.433265 
+  preprint_citation: doi:10.1101/2021.02.28.433265
+  journal_citation: doi:10.7554/eLife.70780
```

Comment:
DOI URL redirects to https://apps.crossref.org/pendingpub/pendingpub.html?doi=10.7554%2FeLife.70780:

![image](https://user-images.githubusercontent.com/1117703/165137767-37d8af66-1409-4abb-92b9-ca85def95917.png)

First time I've ever seen this. The CSL Data isn't complete yet:

```yaml
- id: PYbwVCJ0
  type: webpage
  abstract: Pending Publication
  language: en-GB
  note: |-
    DOI: 10.5555/12345678
    This CSL Item was generated by Manubot v0.5.3 from its persistent identifier (standard_id).
    standard_id: doi:10.7554/elife.70780
  title: pendingpublications
  URL: https://doi.org/gpzrr8
  accessed:
    date-parts:
    - - '2022'
      - 4
      - 25
  DOI: 10.7554/elife.70780
```

Should be fine to merge this now and the correct metadata will eventually be retrieved once the journal updates it.

Reply:
Thank you very much! Yes, weird DOI practice but should be (hopefully) fine.